### PR TITLE
[multi asic] multi asic changes for iface name tests

### PR DESCRIPTION
**Pull request by arlakshm**
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/Azure/SONiC/blob/gh-pages/CONTRIBUTING.md

Please provide following information to help code review process a bit easier:
-->
### Description of PR
<!--
- Please include a summary of the change and which issue is fixed.
- Please also include relevant motivation and context. Where should reviewer start? background context?
- List any dependencies that are required for this change.
-->

Summary:
Fixes # (issue)

### Type of change

<!--
- Fill x for your type of change.
- e.g.
- [x] Bug fix
-->

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ x] Test case(new/improvement)

### Approach
#### What is the motivation for this PR?
Change the tests in the file iface_namingmode.py to support multi asic

#### How did you do it?
Update the CLIs and redis-cmd to support multi asic platforms

#### How did you verify/test it?
**Run on single vs platform**
```
arlakshm@026fa9f03210:/data/public/server14/sonic-mgmt/tests$ ./run_tests.sh -n vms-kvm-t0 -d vlab-01 -c iface_namingmode/test_iface_namingmode.py -f vtestbed.csv -i veos_vtb -u -e '--skip_sanity' -l WARNING                                                                                                                                                                 
=== Running tests in groups ===                                                                                                                                                         
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in a future release.                                                                                                                  
from cryptography.exceptions import InvalidSignature                                                   
=================================== test session starts ====================================                                                                                            
platform linux2 -- Python 2.7.17, pytest-4.6.5, py-1.9.0, pluggy-0.13.1                                                                                                                 
ansible: 2.8.12                                                                                                                                                                         
rootdir: /data/public/server14/sonic-mgmt/tests, inifile: pytest.ini                                                                                                                    
plugins: forked-1.3.0, xdist-1.28.0, repeat-0.8.0, metadata-1.10.0, html-1.22.1, ansible-2.2.2                                                                                          
collected 60 items

                                                                                                                                                                                                                                                                                                                                                            
iface_namingmode/test_iface_namingmode.py::TestShowLLDP::test_show_lldp_table[alias-vlab-01] PASSED [  1%]                                                                              
iface_namingmode/test_iface_namingmode.py::TestShowLLDP::test_show_lldp_neighbor[alias-vlab-01] PASSED [  3%]                                                                           
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_counter[alias-vlab-01] PASSED [  5%]                                                                
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_description[alias-vlab-01] PASSED [  6%]                                                            
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_status[alias-vlab-01] PASSED [  8%]                                                                 
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_portchannel[alias-vlab-01] PASSED [ 10%]                                                            
iface_namingmode/test_iface_namingmode.py::test_show_pfc_counters[alias-vlab-01] PASSED [ 11%]                                                                                          
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_persistent_watermark_headroom[alias-vlab-01] PASSED [ 13%]                                   
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_persistent_watermark_shared[alias-vlab-01] PASSED [ 15%]                                     
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_watermark_headroom[alias-vlab-01] PASSED [ 16%]                                              
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_watermark_shared[alias-vlab-01] PASSED [ 18%]                                                
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_counters[alias-vlab-01] PASSED [ 20%]                                                                         
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_counters_interface[alias-vlab-01] PASSED [ 21%]                                                               
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_persistent_watermark_multicast[alias-vlab-01] PASSED [ 23%]                                                   
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_persistent_watermark_unicast[alias-vlab-01] PASSED [ 25%]                                                     
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_watermark_multicast[alias-vlab-01] PASSED [ 26%]                                                              
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_watermark_unicast[alias-vlab-01] PASSED [ 28%]                                                                
iface_namingmode/test_iface_namingmode.py::TestShowVlan::test_show_vlan_brief[alias-vlab-01] PASSED [ 30%]                                                                              
iface_namingmode/test_iface_namingmode.py::TestShowVlan::test_show_vlan_config[alias-vlab-01] PASSED [ 31%]                                                                             
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_ip[alias-vlab-01] SKIPPED [ 33%]                                                                  
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_state[alias-vlab-01] SKIPPED [ 35%]                                                               
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_speed[alias-vlab-01] SKIPPED [ 36%]
iface_namingmode/test_iface_namingmode.py::test_show_acl_table[alias-vlab-01] SKIPPED [ 38%]
iface_namingmode/test_iface_namingmode.py::test_show_interfaces_neighbor_expected[alias-vlab-01] SKIPPED [ 40%]
iface_namingmode/test_iface_namingmode.py::TestNeighbors::test_show_arp[alias-vlab-01] SKIPPED [ 41%]
iface_namingmode/test_iface_namingmode.py::TestNeighbors::test_show_ndp[alias-vlab-01] SKIPPED [ 43%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_interface[alias-vlab-01] SKIPPED [ 45%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ipv6_interface[alias-vlab-01] SKIPPED [ 46%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_route_v4[alias-vlab-01] SKIPPED [ 48%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_route_v6[alias-vlab-01] SKIPPED [ 50%]
ifaiface_namingmode/test_iface_namingmode.py::TestShowLLDP::test_show_lldp_table[alias-vlab-01] PASSED                                                                           [ 51%]
iface_namingmode/test_iface_namingmode.py::TestShowLLDP::test_show_lldp_neighbor[default-vlab-01] PASSED                                                                         [ 53%]
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_counter[default-vlab-01] PASSED                                                              [ 55%]
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_description[default-vlab-01] PASSED                                                          [ 56%]
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_status[default-vlab-01] PASSED [ 58%]
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_portchannel[default-vlab-01] PASSED [ 60%]
iface_namingmode/test_iface_namingmode.py::test_show_pfc_counters[default-vlab-01] PASSED [ 61%]
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_persistent_watermark_headroom[default-vlab-01] PASSED [ 63%]
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_persistent_watermark_shared[default-vlab-01] PASSED                                   [ 65%]
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_watermark_headroom[default-vlab-01] PASSED                                            [ 66%]
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_watermark_shared[default-vlab-01] PASSED                                              [ 68%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_counters[default-vlab-01] PASSED                                                                       [ 70%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_counters_interface[default-vlab-01] PASSED                                                             [ 71%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_persistent_watermark_multicast[default-vlab-01] PASSED                                                 [ 73%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_persistent_watermark_unicast[default-vlab-01] PASSED                                                   [ 75%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_watermark_multicast[default-vlab-01] PASSED                                                            [ 76%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_watermark_unicast[default-vlab-01] PASSED                                                              [ 78%]
iface_namingmode/test_iface_namingmode.py::TestShowVlan::test_show_vlan_brief[default-vlab-01] PASSED                                                                            [ 80%]
iface_namingmode/test_iface_namingmode.py::TestShowVlan::test_show_vlan_config[default-vlab-01] PASSED                                                                           [ 81%]
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_ip[default-vlab-01] SKIPPED                                                                [ 83%]
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_state[default-vlab-01] SKIPPED                                                             [ 85%]
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_speed[default-vlab-01] SKIPPED                                                             [ 86%]
iface_namingmode/test_iface_namingmode.py::test_show_acl_table[default-vlab-01] SKIPPED                                                                                          [ 88%]
iface_namingmode/test_iface_namingmode.py::test_show_interfaces_neighbor_expected[default-vlab-01] SKIPPED                                                                       [ 90%]
iface_namingmode/test_iface_namingmode.py::TestNeighbors::test_show_arp[default-vlab-01] SKIPPED                                                                                 [ 91%]
iface_namingmode/test_iface_namingmode.py::TestNeighbors::test_show_ndp[default-vlab-01] SKIPPED                                                                                 [ 93%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_interface[default-vlab-01] SKIPPED                                                                           [ 95%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ipv6_interface[default-vlab-01] SKIPPED                                                                         [ 96%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_route_v4[default-vlab-01] SKIPPED                                                                            [ 98%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_route_v6[default-vlab-01] SKIPPED                                                                            [100%]

-------------------------------------------------------- generated xml file: /data/public/server14/sonic-mgmt/tests/logs/tr.xml --------------------------------------------------------
=============================================================================== short test summary info ================================================================================
SKIPPED [6] /data/public/server14/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:589: Unsupported topology
SKIPPED [2] /data/public/server14/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:767: Unsupported topology
SKIPPED [8] /data/public/server14/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:850: Unsupported topology
SKIPPED [4] /data/public/server14/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:791: Unsupported topology
SKIPPED [2] /data/public/server14/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:743: Unsupported topology
======================================================================= 38 passed, 22 skipped in 798.66 seconds ========================================================================
```

**Run on multi asic platform**

```
arlakshm@b65425d3c0c1:/data/repos/public/sonic-mgmt/tests$./run_tests.sh -d multi_asic_1 -n vms13-t1-lag-2 -c iface_namingmode/test_iface_namingmode.py -i /data/repos/files/veos_vtb -f /data/repos/files/vtestbed.csv -l WARNING -k debug -u -m individual -t t1,any -e '--deep_clean --disable_loganalyzer --skip_sanity '                                                                                                        === Running tests individually ===                                                                                                                                                      /usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in a future release.                                                                                                                  from cryptography.exceptions import InvalidSignature                                                   
================================================================================= test session starts ==================================================================================
platform linux2 -- Python 2.7.17, pytest-4.6.5, py-1.9.0, pluggy-0.13.1
ansible: 2.8.12
rootdir: /data/repos/public/sonic-mgmt/tests, inifile: pytest.ini
plugins: forked-1.3.0, xdist-1.28.0, html-1.22.1, metadata-1.10.0, repeat-0.9.1, ansible-2.2.2
collected 60 items

iface_namingmode/test_iface_namingmode.py::TestShowLLDP::test_show_lldp_table[alias-multi_asic_1] PASSED                                                                      [  1%]
iface_namingmode/test_iface_namingmode.py::TestShowLLDP::test_show_lldp_neighbor[alias-multi_asic_1] PASSED                                                                   [  3%]
 iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_counter[alias-multi_asic_1] PASSED                                                       [  5%]
 iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_description[alias-multi_asic_1] PASSED                                                   [  6%]
 iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_status[alias-multi_asic_1] PASSED                                                        [  8%]
 iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_portchannel[alias-multi_asic_1] PASSED                                                   [ 10%]
 iface_namingmode/test_iface_namingmode.py::test_show_pfc_counters[alias-multi_asic_1] PASSED                                                                                 [ 11%]
 iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_persistent_watermark_headroom[alias-multi_asic_1] SKIPPED                         [ 13%]
 iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_persistent_watermark_shared[alias-multi_asic_1] SKIPPED                           [ 15%]
 iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_watermark_headroom[alias-multi_asic_1] SKIPPED                                    [ 16%]
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_watermark_shared[alias-multi_asic_1] SKIPPED                                       [ 18%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_counters[alias-multi_asic_1] SKIPPED                                                                [ 20%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_counters_interface[alias-multi_asic_1] SKIPPED                                                      [ 21%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_persistent_watermark_multicast[alias-multi_asic_1] SKIPPED                                          [ 23%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_persistent_watermark_unicast[alias-multi_asic_1] SKIPPED                                            [ 25%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_watermark_multicast[alias-multi_asic_1] SKIPPED                                                     [ 26%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_watermark_unicast[alias-multi_asic_1] SKIPPED                                                       [ 28%]
iface_namingmode/test_iface_namingmode.py::TestShowVlan::test_show_vlan_brief[alias-multi_asic_1] SKIPPED                                                                     [ 30%]
iface_namingmode/test_iface_namingmode.py::TestShowVlan::test_show_vlan_config[alias-multi_asic_1] SKIPPED                                                                    [ 31%]
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_ip[alias-multi_asic_1] SKIPPED                                                          [ 33%]
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_state[alias-multi_asic_1] PASSED                                                        [ 35%]
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_speed[alias-multi_asic_1] PASSED                                                        [ 36%]
iface_namingmode/test_iface_namingmode.py::test_show_acl_table[alias-multi_asic_1] SKIPPED                                                                                    [ 38%]
iface_namingmode/test_iface_namingmode.py::test_show_interfaces_neighbor_expected[alias-multi_asic_1] SKIPPED                                                                 [ 40%]
iface_namingmode/test_iface_namingmode.py::TestNeighbors::test_show_arp[alias-multi_asic_1] SKIPPED                                                                           [ 41%]
iface_namingmode/test_iface_namingmode.py::TestNeighbors::test_show_ndp[alias-multi_asic_1] SKIPPED                                                                           [ 43%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_interface[alias-multi_asic_1] SKIPPED                                                                     [ 45%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ipv6_interface[alias-multi_asic_1] SKIPPED                                                                   [ 46%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_route_v4[alias-multi_asic_1] SKIPPED                                                                      [ 48%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_route_v6[alias-multi_asic_1] SKIPPED                                                                      [ 50%]
iface_namingmode/test_iface_namingmode.py::TestShowLLDP::test_show_lldp_table[default-multi_asic_1] PASSED                                                                    [ 51%]
iface_namingmode/test_iface_namingmode.py::TestShowLLDP::test_show_lldp_neighbor[default-multi_asic_1] PASSED                                                                 [ 53%]
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_counter[default-multi_asic_1] PASSED                                                      [ 55%]
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_description[default-multi_asic_1] PASSED                                                  [ 56%]
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_status[default-multi_asic_1] PASSED                                                       [ 58%]
iface_namingmode/test_iface_namingmode.py::TestShowInterfaces::test_show_interfaces_portchannel[default-multi_asic_1] PASSED                                                  [ 60%]
iface_namingmode/test_iface_namingmode.py::test_show_pfc_counters[default-multi_asic_1] PASSED                                                                                [ 61%]
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_persistent_watermark_headroom[default-multi_asic_1] SKIPPED                        [ 63%]
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_persistent_watermark_shared[default-multi_asic_1] SKIPPED                          [ 65%]
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_watermark_headroom[default-multi_asic_1] SKIPPED                                   [ 66%]
iface_namingmode/test_iface_namingmode.py::TestShowPriorityGroup::test_show_priority_group_watermark_shared[default-multi_asic_1] SKIPPED                                     [ 68%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_counters[default-multi_asic_1] SKIPPED                                                              [ 70%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_counters_interface[default-multi_asic_1] SKIPPED                                                    [ 71%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_persistent_watermark_multicast[default-multi_asic_1] SKIPPED                                        [ 73%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_persistent_watermark_unicast[default-multi_asic_1] SKIPPED                                          [ 75%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_watermark_multicast[default-multi_asic_1] SKIPPED                                                   [ 76%]
iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_watermark_unicast[default-multi_asic_1] SKIPPED                                                     [ 78%]
iface_namingmode/test_iface_namingmode.py::TestShowVlan::test_show_vlan_brief[default-multi_asic_1] SKIPPED                                                                   [ 80%]
iface_namingmode/test_iface_namingmode.py::TestShowVlan::test_show_vlan_config[default-multi_asic_1] SKIPPED                                                                  [ 81%]
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_ip[default-multi_asic_1] SKIPPED                                                        [ 83%]
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_state[default-multi_asic_1] PASSED                                                      [ 85%]
iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_speed[default-multi_asic_1] PASSED                                                      [ 86%]
iface_namingmode/test_iface_namingmode.py::test_show_acl_table[default-multi_asic_1] SKIPPED                                                                                  [ 88%]
iface_namingmode/test_iface_namingmode.py::test_show_interfaces_neighbor_expected[default-multi_asic_1] SKIPPED                                                               [ 90%]
iface_namingmode/test_iface_namingmode.py::TestNeighbors::test_show_arp[default-multi_asic_1] SKIPPED                                                                         [ 91%]
iface_namingmode/test_iface_namingmode.py::TestNeighbors::test_show_ndp[default-multi_asic_1] SKIPPED                                                                         [ 93%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_interface[default-multi_asic_1] SKIPPED                                                                   [ 95%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ipv6_interface[default-multi_asic_1] SKIPPED                                                                 [ 96%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_route_v4[default-multi_asic_1] SKIPPED                                                                    [ 98%]
iface_namingmode/test_iface_namingmode.py::TestShowIP::test_show_ip_route_v6[default-multi_asic_1] SKIPPED                                                                    [100%]

--------------------------------------- generated xml file: /data/repos/public/sonic-mgmt/tests/logs/iface_namingmode/test_iface_namingmode.xml ----------------------------------------
=============================================================================== short test summary info ================================================================================SKIPPED [2] /data/repos/public/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:771: CLI command not supported
SKIPPED [2] /data/repos/public/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:625: No L3 physical interface present
SKIPPED [4] /data/repos/public/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:525: Unsupported topology
SKIPPED [8] /data/repos/public/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:350: CLI command not supported
SKIPPED [8] /data/repos/public/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:853: No non-portchannel member interface present
SKIPPED [2] /data/repos/public/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:746: No non-portchannel member interface present
SKIPPED [4] /data/repos/public/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:794: CLI not supported
SKIPPED [12] /data/repos/public/sonic-mgmt/tests/iface_namingmode/test_iface_namingmode.py:423: CLI command not supported
======================================================================= 18 passed, 42 skipped in 729.32 seconds ========================================================================+ exit 0
arlakshm@b65425d3c0c1:/data/repos/public/sonic-mgmt/tests$
```
#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation 
<!--
(If it's a new feature, new test case)
Did you update documentation/Wiki relevant to your implementation?
Link to the wiki page?
-->

### CODE REVIEW

**Comment by arlakshm** [comment]
@ wangxin can you review this PR

File: tests/common/devices/multi_asic.py [hunk 1]
```diff
@@ -286,6 +286,18 @@ def is_service_running(self, service_name, docker_name=None):
                 return False
 
         return True
+    
+    def get_asic_index_for_port(self, portname):
```
Comment (smaheshm) [inline tests/common/devices/multi_asic.py:290]: can you reuse this:

https://github.com/Azure/sonic-mgmt/blob/7baf71175fdfd696de48fd5ad58f04eac11c2129/tests/common/devices/multi_asic.py#L290

File: tests/iface_namingmode/test_iface_namingmode.py [hunk 2]
```diff
@@ -48,14 +48,20 @@ def setup(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo):
...
         port_speed[port_alias_new] = port_speed_facts[port_alias_old]
 
+        #sonic-db-cli command
+        db_cmd = 'sudo {} CONFIG_DB HSET "PORT|{}" alias {}'\
+            .format(duthost.asic_instance(asic_index).sonic_db_cli,
+                    item,
+                    port_alias_new)
         # Update port alias name in redis db
-        duthost.command('redis-cli -n 4 HSET "PORT|{}" alias {}'.format(item, port_alias_new))
+        duthost.command(db_cmd)
```
Comment (smaheshm) [inline tests/iface_namingmode/test_iface_namingmode.py:64]: there is a "sonic_asic.run_redis_cli_cmd". But this is fine too.

File: tests/iface_namingmode/test_iface_namingmode.py [hunk 3]
```diff
@@ -326,6 +342,12 @@ def test_show_pfc_counters(setup, setup_config_mode):
 
 class TestShowPriorityGroup():
 
+    @pytest.fixture(scope="class", autouse=True)
+    def setup_check_topo(self, duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+        duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+        if duthost.is_multi_asic:
+            pytest.skip('CLI command not supported')
```
Comment (smaheshm) [inline tests/iface_namingmode/test_iface_namingmode.py:349]: may be add "CLI command not supported on Multi-ASIC". Is it possible to add what command is not supported.

File: tests/iface_namingmode/test_iface_namingmode.py [hunk 4]
```diff
@@ -390,8 +412,15 @@ def test_show_priority_group_watermark_shared(self, setup, setup_config_mode):
             for intf in setup['up_ports']:
                 assert re.search(r'{}.*'.format(intf), show_pg) is not None
 
+
 class TestShowQueue():
 
+    @pytest.fixture(scope="class", autouse=True)
```
Comment (smaheshm) [inline tests/iface_namingmode/test_iface_namingmode.py:418]: Looks same as above. Define it outside the class?

File: tests/iface_namingmode/test_iface_namingmode.py (hunk 3 above)
Comment (smaheshm) [reply tests/iface_namingmode/test_iface_namingmode.py:349]: I guess the CLI command can be determined from the test name.

File: tests/iface_namingmode/test_iface_namingmode.py [hunk 5]
```diff
@@ -709,14 +757,18 @@ def test_show_acl_table(setup, setup_config_mode, tbinfo):
...
-def test_show_interfaces_neighbor_expected(setup, setup_config_mode, tbinfo):
+def test_show_interfaces_neighbor_expected(setup, setup_config_mode, tbinfo,duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     """
     Checks whether 'show interfaces neighbor expected' lists the
     interface names as per the configured naming mode
     """
     if tbinfo['topo']['type'] not in ['t1', 't2']:
         pytest.skip('Unsupported topology')
 
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
```
Comment (smaheshm) [inline tests/iface_namingmode/test_iface_namingmode.py:768]: can you use the new fixture you wrote to skip masic patform.

File: tests/common/devices/multi_asic.py (hunk 1 above)
Comment (arlakshm) [reply tests/common/devices/multi_asic.py]: changed in the latest comment

File: tests/iface_namingmode/test_iface_namingmode.py (hunk 2 above)
Comment (arlakshm) [reply tests/iface_namingmode/test_iface_namingmode.py:69]: Using `sonic_db_cli ` here to operate on DB entries in namespace is easier

File: tests/iface_namingmode/test_iface_namingmode.py (hunk 3 above)
Comment (arlakshm) [reply tests/iface_namingmode/test_iface_namingmode.py]: Yes, the testcase name will give the failed CLI, so not adding the CLI command in the message

File: tests/iface_namingmode/test_iface_namingmode.py (hunk 4 above)
Comment (arlakshm) [reply tests/iface_namingmode/test_iface_namingmode.py:421]: The fixture must be defined in the class to skip all the tests in the class. The logic to check if the tests need to be skipped is moved to common function

File: tests/iface_namingmode/test_iface_namingmode.py (hunk 5 above)
Comment (arlakshm) [reply tests/iface_namingmode/test_iface_namingmode.py]: Changed to use common function

**Review by smaheshm — APPROVED** [review]
:shipit:

**Pull request merged**